src/modules/auth/reducers.js

```diff
@@ -0,0 +1,10 @@
+import createReducer from '../../utils/create-reducer'
+import { AUTHENTICATE_WITH_GOOGLE } from './actions'
+
+const INITIAL_STATE = null
+
+const token = createReducer(INITIAL_STATE, {
+  [AUTHENTICATE_WITH_GOOGLE.FULFILLED]: (state, { payload }) => payload,
+})
+
+export default token
```

Comment:
We usually export reducer as `const` so we might have more than one reducer per file. If you are sure that's gonna be just one reducer maybe just rename this file to `reducer.js`